code/script.js

```diff
@@ -29,19 +42,152 @@ const showMessage = (message, sender) => {
+  }
+}
 
 // Starts here
+// When website loaded, chatbot asks first question.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello gorgeous! What is your name?`, 'bot')
+}
+const reply1 = (message) => {
+  questionNumber++ 
+  botAnswer(`Hey ${message}, Which booking would you like to make? `)
+ 
+  inputWrapper.innerHTML = `
+     <button id="cutBtn" >Hair cut</button>
+     <button id="colorBtn" >Hair color</button>
+     <button id="spaBtn">Hair spa</button>`
+     
+   document.getElementById('cutBtn') .addEventListener('click', () => nextQuestion('Hair cut'))
+   document.getElementById('colorBtn').addEventListener('click', () => nextQuestion('Hair color'))
+   document.getElementById('spaBtn').addEventListener('click', () => nextQuestion('Hair spa'))
+ }
+ 
+ const reply2 = (message) => {
+   questionNumber ++ 
+   if (message === 'Hair cut') {
+    botAnswer(`Choose a Stylist below`)
+    inputWrapper.innerHTML = `
+    <select name="stylist" id="select">
+     <option value="" selected disabled> Choose your options</option>
+     <option value="Junior stylist">Junior stylist</option>
+     <option value="Senior stylist">Senior Stylist</option>  
+    </select>`
+ 
+ const select = document.getElementById('select')
+ select.addEventListener('change', () => nextQuestion(select.value))
 }
+ 
+   if (message === 'Hair color') {
+    botAnswer(`Great! Choose your desired style`)
+    inputWrapper.innerHTML = `
+    <select name="colors" id="select">
+     <option value="" selected disabled> Choose your options</option>
+     <option value="Baylage">Baylage</option>
+     <option value="Highlights">Highlights</option>
+     <option value="Ombré">Ombré </option>
+     <option value="Streaks">Streaks </option>
+     <option value="Others">Others </option>
+    </select>`
+
+  select.addEventListener('change', () => nextQuestion(select.value))
+ }
+ 
+   else if (message === 'Hair spa') {
+     botAnswer(`Great! Choose your desired treatment`)
+     inputWrapper.innerHTML = `
+    <select name="spa" id="select">
+     <option value="" selected disabled> Choose your options</option>
+     <option value="Oil treatment">Oil treatment</option>
+     <option value="Hair wash with blow out">Hair wash with blow out </option>
+     <option value="L'oréal">L'oréal </option>
+     <option value="Olaplex">Olaplex </option>
+   </select>`
+
+   select.addEventListener('change', () => nextQuestion(select.value))
+ }  
+  }
+
+  const reply3 = (message) => {
+    questionNumber ++ 
+
+      botAnswer(`You will look amazing! Now choose the date and time for your ${message}.`)
+      inputWrapper.innerHTML = `
+      <label for="appointment"></label>
+      <input type="datetime-local" id="appointment" name="appointment">
+      `
+      document.getElementById('appointment').addEventListener('change', () => nextQuestion(appointment.value))
+     
+  }
+
```

Comment:
Since you use rather "non-descriptive" names such as reply4 etc, maybe you should add comments before each section to make it easier to follow? For instance *//Booking confirmation (just to make it easier to follow)